analysis/report/entrypoint.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+# params:
+#  <IPFS_API> . i.e. /ip4/x.x.x.x/tcp/5001
+# this can be passed as an argument or environment variable.
+#
+# 1. Generate reports
+# 2. Pin to IPFS
+# 3. Copy to MFS
+# 4. Pubish to IPNS
+
+if [[ ! $# -ne 1  ]]
+then
+	IPFS=$1
+elif [[ ! -z "${IPFS_API}" ]]
+then
+	IPFS="${IPFS_API}"
+else
+	echo "Sorry, I don't know where to upload reports. This is a configuration error. Will not continue." >&2
+	exit 1
+fi
+
+
+# this is the directory where reports will be generated.
+mkdir reports
+
+# Generate report
+poetry run python ./gen_report.py || echo "Error generating reports. I'll publish what I got." && echo "could not generate the report today" > reports/report-error
```

Comment:
Just a suggestion, it may be helpful to publish the log output in case of errors. That makes debugging problems easier in the future.

Right now the report errors just contain `could not generate the report today`: 
https://ipfs.io/ipns/k51qzi5uqu5dhic4e8njzedawtn6oklphhr9x5p6b8a2wzc56chdfjqtaddmr8/nebula-22-01-09/report-error